core/BUILD.bazel

```diff
@@ -1,33 +1,23 @@
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("//docs:stardoc.bzl", "generate")
 
 package(default_visibility = ["//visibility:public"])
 
-exports_files([
-    "nixpkgs.bzl",
-])
-
 filegroup(
     name = "srcs",
     srcs = glob(["**"]),
     visibility = ["//visibility:public"],
 )
 
-bzl_library(
-    name = "bazel_tools",
-    srcs = [
-        "@bazel_tools//tools:bzl_srcs",
-    ],
-)
-
-bzl_library(
```

Comment:
The `bzl_library` targets are needed for other rule sets depending on `rules_nixpkgs_*`.

E.g. `rules_haskell` generates its API docs using Stardoc, too. But, Startdoc requires all transitively loaded `.bzl` files to be declared `bzl_library` dependencies. So, to generate `rules_haskell` API docs, we'll need the `rules_nixpkgs` `bzl_library` targets.

Reply:
I see that the `bzl_library` is generated as part of the docs `generate` macro. As mentioned in https://github.com/tweag/rules_nixpkgs/pull/202#discussion_r831945136, we'll need the `bzl_library` targets in a BUILD file that does not depend on Startdoc, so that down-stream (e.g. rules_haskell) can define its own `bzl_library` targets, without depending on dev-dependencies of `rules_nixpkgs_*`.

Reply:
I was under the impression that those `bzl_library` targets are _only_ used for documentation. Files like `@rules_nixpkgs_core//:nixpkgs.bzl` are exposed by filegroup `:srcs` if I understand correctly. One can still run tests and examples when I delete all calls to `generate()`. Also I can't find where `rules_haskell` may depend on the `nixpkgs` `bzl_library`.